plugin/CactbotEventSource/WipeDetector.cs

```diff
@@ -24,10 +24,13 @@ class WipeDetector {
         if (log.IndexOf(" 00:0038:cactbot wipe", StringComparison.Ordinal) != -1) {
           WipeIt();
         }
+        if (log.IndexOf(" 00:0038:end", StringComparison.Ordinal) != -1) {
```

Comment:
This looks good to me!

(The wipe detector used to have a lot more logic (looking at hps and seeing if you come back to life without limit break or raise) but is much simpler these days.  In my mind this is all a bit vestigial and probably we should just do this in overlays that care about it rather than having a special event.  I think this is fine for now, this is all just musing about code that kind of doesn't make a ton of sense anymore.)

Reply:
So, finally got around to testing this, and it seems like WipeDetector just flat out doesn't work for any of the three wipe types. It's likely completely broken, but with no way to build locally I therefore can't debug locally to verify. I did test all three types and nothing triggered the `onPartyWipe` event.

Instead, it probably makes sense to just move this logic to `popup-text.ts:866` where it's already checking for `cactbot wipe`, as well as relevant updates in oopsy's `damage_tracker.ts` and pullcounter's `pullcounter.ts`, then remove the cactbot c# event entirely? If that's the case, I'll close this PR and open that as a different PR since it would have a different scope.

Reply:
Hmm!  Maybe it never got updated for log line format changes and nobody ever noticed.  I think removing the C# makes a lot of sense and moving it all to the relevant ts code.